Kitodo/src/main/java/de/sub/goobi/config/Parameters.java

```diff
@@ -0,0 +1,695 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package de.sub.goobi.config;
```

Comment:
New files should be inside `org.kitodo` and this file could go to `org.kitodo.config` package realm.